### PR TITLE
Added New Navbar and Config 

### PR DESCRIPTION
**Pull request by biratdatta**
Created a New Navbar Component and a config file that contains the features and default value.

image
image

@prtkjakhar and @geeky-abhishek check this PR

### CODE REVIEW

**Comment by geeky-abhishek** [comment]
Don't replace the existing navbar it is used for the project , create a molecule and display it under /components.
[Refrence](https://mui.com/material-ui/react-app-bar/)

File: src/components/navbar/config.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (geeky-abhishek) [inline src/components/navbar/config.json]: it should be like
```json
 {
  "component": {
    "navbar": {
      // ...componentProps
    }
  }
}
```

File: src/components/navbar/index.tsx [hunk 2]
```diff
@@ -1,97 +1,83 @@
...
-                <Sidebar isOpen={isDrawerOpen} onToggle={onDrawerToggle} />
-            </Box>
-        </>
-    )
-}
+        
+          {showRightLogos && (
+            <div>
+              {rightLogoIcons.map((logo, index) => (
+                <img key={index} src={logo} alt={`Right Logo ${index}`} style={{ maxHeight: '48px' }} />
```
Comment (geeky-abhishek) [inline src/components/navbar/index.tsx]: don't use index as a key

**Comment by biratdatta** [comment]
@geeky-abhishek Changed the code to resolve the current issues. If there are any further adjustments or if you have additional feedback, please let me know.

**Pull request merged**